src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![doc(html_root_url = "https://docs.rs/crate/handlebars/")]
+#![doc(html_root_url = "https://docs.rs/handlebars/CrateVersion")]
```

Comment:
There is an issue if you hard-coded `CrateVersion` here that once it's replaced in a release. There will be no such `CrateVersion` to replace in future release. You can use regex to capture the version number here.

There is an example in cargo-release's release.toml as example. Replying from mobile, sorry for unable to link it.

Reply:
I've fixed in e6de93a.